### PR TITLE
5.9 release update

### PR DESCRIPTION
**Pull request by jsiemensLatitudegeo**
Updated VertiGIS.Mobile package to 5.9. PR build for now. Will update to final package version before completing merge.
Updated other packages to match VergiGIS.Mobile 5.9 dependencies.
Updated layout.xml files to make use of expand component.
Updated layout style attributes to use "round" instead of "map-widget".

### CODE REVIEW

File: Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/App/GeocortexMobileViewer/layout-large.xml [hunk 1]
```diff
@@ -3,12 +3,12 @@
     xmlns="https://geocortex.com/layout/v1"
     xmlns:gxm="https://geocortex.com/layout/mobile/v1"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="https://geocortex.com/layout/v1 ../../../../../ViewerSpec/layout/layout-mobile.xsd">
+    xsi:schemaLocation="https://geocortex.com/layout/v1 ../../ViewerSpec/layout/layout-mobile.xsd">
```
Comment (ianschmitz) [inline Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/App/GeocortexMobileViewer/layout-large.xml:6]: Maybe an accidental change?

**Review by ianschmitz — APPROVED** [review]

File: Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/App/GeocortexMobileViewer/layout-large.xml [hunk 2]
```diff
@@ -41,9 +46,9 @@
 
     <!-- Taskbar / layers -->
     <layer-list id="layers"/>
-
-    <!-- Taskbar / legend -->
-    <legend id="legend"/>
+    
+    <!-- Basmeap picker -->
```
Comment (kewalker) [inline Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/App/GeocortexMobileViewer/layout-large.xml:50]: Typo.

File: Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/Layout/ButtonStyles/README.md [hunk 3]
```diff
@@ -5,10 +5,10 @@ This sample demonstrates the Geocortex Mobile button styles that are available.
...
-  - map-widget: text
-  - icon and text
-  - map-widget: icon and text
+  - square (default): icon
+  - round: icon
+  - square (default): text
+  - round: text
+  - square (default): icon and text
+  - round: icon and text
 - Map Widget styles are commonly used for buttons placed on the map.
```
Comment (kewalker) [inline Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/Layout/ButtonStyles/README.md:14]: Round?

File: Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/Layout/ButtonStyles/button-styles.xml [hunk 4]
```diff
@@ -10,9 +10,9 @@
   <panel>
     <stack title="Button Styles" halign="start">
       <text margin="1">Map Widget Buttons</text>
```
Comment (kewalker) [inline Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/Layout/ButtonStyles/button-styles.xml:12]: Round?

**Review by kewalker — APPROVED** [review]

File: Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/App/GeocortexMobileViewer/layout-large.xml (hunk 1 above)
Comment (kewalker) [reply Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/App/GeocortexMobileViewer/layout-large.xml:6]: Looks like this change is correct. These files weren't updated when being copied/pasted when the project was created (the app.json was). Resolving.

File: Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/App/GeocortexMobileViewer/layout-large.xml (hunk 2 above)
Comment (kewalker) [reply Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/App/GeocortexMobileViewer/layout-large.xml]: Fixed.

File: Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/Layout/ButtonStyles/README.md (hunk 3 above)
Comment (kewalker) [reply Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/Layout/ButtonStyles/README.md]: Updated language.

File: Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/Layout/ButtonStyles/button-styles.xml (hunk 4 above)
Comment (kewalker) [reply Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/Layout/ButtonStyles/button-styles.xml]: Updated language.

**Pull request merged**